src/test/java/tests/MilestonePickerStateTests.java

```diff
@@ -9,46 +9,55 @@
 import java.util.List;
 
 import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 
 public class MilestonePickerStateTests {
     public MilestonePickerState prepareUnassignedState() {
-        List<PickerMilestone> labelList = new ArrayList<>();
-        PickerMilestone label = new PickerMilestone(new TurboMilestone("testrepo", 1, "milestone1"));
-        labelList.add(label);
-        return new MilestonePickerState(labelList);
+        List<PickerMilestone> milestoneList = new ArrayList<>();
+        PickerMilestone milestone = new PickerMilestone(new TurboMilestone("testrepo", 1, "milestone1"));
+        milestoneList.add(milestone);
+        return new MilestonePickerState(milestoneList, "");
     }
 
     public MilestonePickerState prepareAssignedState() {
-        List<PickerMilestone> labelList = new ArrayList<>();
-        PickerMilestone label = new PickerMilestone(new TurboMilestone("testrepo", 1, "milestone1"));
-        PickerMilestone label2 = new PickerMilestone(new TurboMilestone("testrepo", 1, "milestone2"));
-        label2.setSelected(true);
-        labelList.add(label);
-        labelList.add(label2);
-        return new MilestonePickerState(labelList);
+        List<PickerMilestone> milestoneList = new ArrayList<>();
+        PickerMilestone milestone1 = new PickerMilestone(new TurboMilestone("testrepo", 1, "milestone1"));
+        PickerMilestone milestone2 = new PickerMilestone(new TurboMilestone("testrepo", 2, "milestone2"));
+        milestoneList.add(milestone1);
+        milestoneList.add(milestone2);
+        return new MilestonePickerState(milestoneList, milestone2.getTitle());
```

Comment:
Put a header comment to this method to tell that `userInput is set to title of milestone 2` or something similar.